prime-router/src/main/kotlin/cli/tests/TestReportStream.kt

```diff
@@ -516,10 +518,12 @@ abstract class CoolTest {
             val secsElapsed = OffsetDateTime.now().second % 60
             // Wait until the top of the next minute, and pluSecs more, for 'batch', and 'send' to finish.
             var waitSecs = 60 - secsElapsed + plusSecs
-            if (secsElapsed > (60 - plusSecs) || env != ReportStreamEnv.LOCAL) {
+            if (env != ReportStreamEnv.LOCAL) {
+                // We are in Test or Staging, which don't execute on the top of the minute. Hack:
```

Comment:
Instead of adding `Hack:`, consider using `HACKHACK`; this ensures that when you grep for hacks (`HACKHACK`), lines that talk about "getting hacked:..." won't show up

It also means we can easily search for places where we have a 'hack' in place: `grep -Ri HACKHACK *.kt`

Reply:
And as an aside: what is the plan to get rid of this hack? It would make the tests run smoother, and faster, which I'm sure all developers would appreciate;

Can you file a GitHub Issue to address this hack?

Reply:
Changed to 'kluge' ;) 
And good point - I forgot to ticket this.  See #1954